tests/proto3/proto3_compat_test.go

```diff
@@ -52,7 +56,8 @@ func TestFixed32Roundtrip(t *testing.T) {
 }
 
 func TestVarintZigzagRoundtrip(t *testing.T) {
-	// amino varint (int) <-> protobuf zigzag32 (int32)
+	t.Skip("zigzag encoding isn't default anymore for (unsigned) ints")
```

Comment:
TODO add tags and test this ?

Reply:
Added an issue for this: #239 
I'll add a todo and a link to the issue, thanks!